pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
```
import pandas.core.strings as strings
```
use `strings.StringMethods` or, to avoid dict look-up, `StringMethods = strings.StringMethods`